src/main/java/com/microfocus/application/automation/tools/run/AbstractSvRemoteRunner.java

```diff
@@ -21,13 +41,13 @@
 import hudson.model.TaskListener;
 import jenkins.security.MasterToSlaveCallable;
 
-public abstract class AbstractRemoteRunner<T extends AbstractSvRunModel> extends MasterToSlaveCallable<String, Exception> {
+public abstract class AbstractSvRemoteRunner<T extends AbstractSvRunModel> extends MasterToSlaveCallable<String, Exception> {
```

Comment:
Besides the name, I also require that each new class will be in its folder.
For your case it should be under: hpe-application-automation-tools-plugin/src/main/java/com/microfocus/application/automation/tools/sv/
After /sv/ you can define your own package hirearchy 
For example see [Octane's](https://github.com/jenkinsci/hpe-application-automation-tools-plugin/tree/latest/src/main/java/com/microfocus/application/automation/tools/octane) hirearchy.